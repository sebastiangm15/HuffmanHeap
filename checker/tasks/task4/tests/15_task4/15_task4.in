5
5 Z1
6 Z2
7 Z3
8 Z4
9 Z5
3
Z1
Z3
Z5
