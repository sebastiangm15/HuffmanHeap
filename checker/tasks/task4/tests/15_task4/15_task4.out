Z3Z4Z5Z1Z2
